handler/user_handler.go

```
@@ -60,3 +61,68 @@ func LoginHandler(userService service.UserService) http.HandlerFunc {
 		w.Write(responseBytes)
 	}
 }
+
+func CreateUserHandler(userService service.UserService) http.HandlerFunc {
+
+	return func(w http.ResponseWriter, r *http.Request) {
+
+		w.Header().Set("Context-Type", "application/json")
+
+		var req contract.CreateUserRequest
+
+		err := json.NewDecoder(r.Body).Decode(&req)
+
+		if err != nil {
+
+			fmt.Printf("handler: error while decoding request for create user: %s", err.Error())
+
+			w.WriteHeader(http.StatusBadRequest)
+			responseBytes, _ := json.Marshal(contract.ErrorResponse{Error: "invalid request"})
```

don't ignore err with _